### DISCUSSION TITLE
Specific map w/ autopilot = Memory fault(coredump)

### DISCUSSION THREAD

**Discussion by yoolooshoo** [ppy/osu#23562] [Q&A]
### Type

Crash to desktop

### Bug description

Some info:
Map: https://osu.ppy.sh/beatmapsets/1748483#osu/3576473
OS: Fedora 38, kernel 6.2.14-300.fc38.x86_64

1. Launch the above map with autopilot right after the game starts.
2. Have to tap space 2 times to skip intro, weird.
3. When first circles appear, get a crash with message `Memory fault(coredump)`.

Tried a few other maps & mods, they work fine.

### Screenshots or videos

https://github.com/ppy/osu/assets/133593780/72ac0e9b-8c81-4b07-ae94-fb154f65045d

### Version

Lazer 2023.513.0

### Logs

[database.log](https://github.com/ppy/osu/files/11472599/database.log)
[input.log](https://github.com/ppy/osu/files/11472600/input.log)
[network.log](https://github.com/ppy/osu/files/11472601/network.log)
[performance.log](https://github.com/ppy/osu/files/11472603/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11472604/runtime.log)

**Comment by bdach** [comment]
Cannot reproduce here.

Is there something interesting in kernel logs (`dmesg`) that looks related to the crash?

**Comment by yoolooshoo** [comment]
dmesg:
```
[  285.672512] show_signal_msg: 31 callbacks suppressed
[  285.672514] osu![2316]: segfault at 0 ip 00007ffa84c3acf8 sp 00007ffa737fd4e0 error 4 in nouveau_dri.so[7ffa8428d000+11a0000] likely on CPU 1 (core 1, socket 0)
[  285.672523] Code: 8d 52 04 c1 e0 10 4d 89 54 24 30 0d 7c 05 00 60 89 02 85 db 0f 84 b8 03 00 00 41 89 db 31 c0 4a 8d 3c 9d 00 00 00 00 0f 1f 00 <41> 8b 0c 06 89 4c 02 04 48 83 c0 04 48 39 f8 75 ef 4a 8d 04 9d 00
```

**Comment by longnguyen2004** [comment]
> `nouveau_dri.so`

Are you using the open-source NVIDIA driver? If so, can you try the official NVIDIA driver?
Also, does disabling storyboards help?

**Comment by yoolooshoo** [comment]
> If so, can you try the official NVIDIA driver?

No, too much of a hassle with my half-broken system.

> Also, does disabling storyboards help?

Yes, does not crash anymore, but skip still does not work the first time

**Comment by Joehuu** [comment]
> Yes, does not crash anymore, but skip still does not work the first time

That issue is already tracked https://github.com/ppy/osu/issues/9787.

**Comment by bdach** [comment]
I'm not super sure how to even determine if this is us doing anything wrong, or is it a nouveau bug.

@yoolooshoo does this occur if you switch renderers to `OpenGL (Legacy)`?

**Comment by yoolooshoo** [comment]
> does this occur if you switch renderers to `OpenGL (Legacy)`?

No
